ui/components/app/assets/asset-list/asset-list-control-bar/index.scss

```diff
@@ -0,0 +1,12 @@
+.asset-list-control-bar {
+  display: flex;
+  justify-content: space-between;
+  cursor: pointer;
+  padding-top: 16px;
+  margin-left: 16px;
+  margin-right: 16px;
+
+  button:hover {
+    background-color: var(--color-background-hover);
+  }
```

Comment:
Could we use a classname here instead of the element? This will help with discoverability and reduce any issues if we were to use another button element within this component

Reply:
https://github.com/MetaMask/metamask-extension/pull/27084/commits/4baa9ad7f67c1daf8b359232dd7dbbe3902d84a7